group_project_v2/project_api/api_implementation.py

```diff
@@ -6,7 +6,9 @@
 from group_project_v2.api_error import api_error_protect
 from group_project_v2.json_requests import DELETE, GET, PUT, POST
 from group_project_v2.utils import memoize_with_expiration, build_date_field
-from group_project_v2.project_api.dtos import UserDetails, ProjectDetails, WorkgroupDetails, CompletionDetails
+from group_project_v2.project_api.dtos import (
+    UserDetails, ProjectDetails, WorkgroupDetails, CompletionDetails,
+    OrganisationDetails, UserGroupDetails)
```

Comment:
@jbzdak nit: closing parenthesis on new line. Yes, PyCharm never do this when automatically adding import, and yes, I was manually correcting this all the way :)